src/driver.rs

```diff
@@ -1,6 +1,7 @@
-use std::{fs, path::PathBuf};
+use std::{collections::BTreeMap, fs, path::PathBuf};
 
 use ark_ff::AdditiveGroup as _;
+use noirc_driver::gen_abi;
```

Comment:
Do not import this – in this file we call eveyrthing from noir qualified, so that it's obvious when you see a call that it calls into their compiler